src/pages/components/cards.mdx

```diff
@@ -103,13 +103,23 @@ Top-aligning the pictogram should only be used when the pictogram is strongly an
 By default, the entire Card is clickable with a single destination, but by adding the Static modifier you can add multiple links to a single card. This can be useful when you have multiple interactive interactive elements on a single card such as CTAs and Tag links that take the user to different pages. Static cards can appear with or without a border.
```

Comment:
```suggestion
By default, the entire Card is clickable with a single destination, but by adding the Static modifier you can add multiple links to a single card. This can be useful when you have multiple interactive interactive elements on a single card that take the user to different pages. Static cards can appear with or without a border.
```